roles/tests/test_api.py

```diff
@@ -76,6 +79,60 @@ def test_roles_init_new_repo_2(self):
             sorted(RepoPermission.author_permissions())
         )
 
+    def test_roles_init_new_repo_3(self):
```

Comment:
On the other had, I guess it's PEP8 complaining, not Pylint: is there a way to disable these checks?